jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/UpdateChecker.java

```diff
@@ -134,7 +133,7 @@ void setDisableUpdateCheck(boolean disableUpdateCheck) {
       }
 
       // Check time of last update check
-      if (Files.exists(lastUpdateCheck)) {
+      if (Files.exists(lastUpdateCheck) && Files.size(lastUpdateCheck) > 0) {
```

Comment:
But this means if we ever create an empty file, we can never escape the situation and always call the endpoint. I think for this file, we don't need to check the size, because we don't show a warning. And this will auto-fix the empty file problem by deleting it.

Reply:
If we create an empty file won't it be fixed when it's written to at L163? I guess it doesn't matter.

Reply:
Oh! I didn't realize that.